metadata-models/src/test/java/com/linkedin/metadata/ModelValidation.java

```diff
@@ -48,16 +49,20 @@ public void validateSnapshots() throws Exception {
     getClassesInPackage("com.linkedin.metadata.snapshot", IGNORED_SNAPSHOT_CLASSES).forEach(
         SnapshotValidator::validateSnapshotSchema);
 
-    SnapshotValidator.validateUniqueUrn(
-        getClassesInPackage("com.linkedin.metadata.snapshot", IGNORED_SNAPSHOT_CLASSES).collect(Collectors.toList()));
+    @SuppressWarnings("unchecked")
```

Comment:
Is this change related to ES migration? Or, is it only a side effect of bumping to 0.2.13? Assuming it's the latter. Then, better bump master to 0.2.13 first and then branch off from there.

Reply:
Yes, it's the result of bumping up `datahub-gma` version to 0.2.13.
Considering the ES7 changes introduced in this PR will sit in this branch for some time before we merge to `master`, there will be changes in `master` branch corresponding to version bump of `datahub-gma`, so to some extent there will be merge conflicts before this PR is merged to `master`.
Was your concern around merge conflicts or restricting the PR to ES7 changes only?

Reply:
The latter, restricting the PR to ES7 changes only.

Reply:
Sure, we can upgrade `datahub-gma` version and restrict this PR to ES7 changes only.